src/main/java/uk/co/ramp/covid/simulation/covid/Covid.java

```diff
@@ -150,6 +152,7 @@ public CStatus stepInfectionSymptomatic(Time t) {
             status = CStatus.PHASE1;
         } else if ((latentPeriod + p1 + p2) > infCounter) {
             phase2 = true;
+            isSymptomatic = true; 
```

Comment:
We probably need to do a household isolate here (as we do on line 176?). Probably best to make this a function so we always toggle symptoms the same way that makes people (consider if they should) isolate.